pgjdbc/src/main/java/org/postgresql/ds/common/BaseDataSource.java

```diff
@@ -1089,6 +1089,16 @@ public void setUrl(String url) {
     }
   }
 
+  /**
+   * Sets properties from a {@link DriverManager} URL.
+   * Added to follow convention used in other DBMS.
+   *
+   * @param url properties to set
+   */
+  public void setURL(String url) {
```

Comment:
Should not the setURL come in pair with the getURL?

Reply:
I've added getURL.